scripts/compute_package_name.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# ./compute_package_name <CHANNEL>
+
```

Comment:
Should we have another name for testnet packages? Or are we using the same packages and suggesting users swap their genesis files?

Reply:
testnet/mainnet both run the same binaries (the ones from rel/stable)

Reply:
What names do you have in mind?  I'm afraid I don't understand your second question.

Reply:
They should be the same binaries, but should they be the same packages? Users may want the option to install a testnet package that comes with the correct genesis file already set up in their data dir

Reply:
I've talked with Tsachi about this and the feeling is that a user can do that with the tarballs, but different expectations come with downloading from a repo.

For instance, a more experienced user will download the tarballs and have the binaries point to different data directories, but a different kind of user will want to download from a repository and just have it work.

I'd also like to keep the existing implementation as is for now, and consider reviewing the outcome once the entire build system is complete.